sdks/client-csharp/ConsoleApp1/Program.cs

```diff
@@ -28,24 +28,34 @@ static void Main(string[] args)
         Console.WriteLine($"New features");
       };
 
-      var esl = new EventServiceListener();
-
-      esl.Init("http://localhost:8553/features/default/ec6a720b-71ac-4cc1-8da1-b5e396fa00ca/Kps0MAqsGt5QhgmwMEoRougAflM2b8Q9e1EFeBPHtuIF0azpcCXeeOw1DabFojYdXXr26fyycqjBt3pa", fh);
-
-      Console.Write("Press a key");
-      Console.ReadKey();
+      fh.AddAnalyticCollector(new GoogleAnalyticsCollector("UA-example", "1234-5678-abcd-abcd",
+        new GoogleAnalyticsHttpClient()));
 
-      Console.Write("Press a key (changed context)");
-
-      fh.ClientContext.UserKey("DJElif").Country(StrategyAttributeCountryName.Turkey).Attr("city", "istanbul").Build();
-      Console.ReadKey();
+      var esl = new EventServiceListener();
 
-      Console.Write("Press a key (change context2)");
-      Console.ReadKey();
+      esl.Init("http://localhost:8553/features/default/ce6b5f90-2a8a-4b29-b10f-7f1c98d878fe/VNftuX5LV6PoazPZsEEIBujM4OBqA1Iv9f9cBGho2LJylvxXMXKGxwD14xt2d7Ma3GHTsdsSO8DTvAYF", fh);
 
-      fh.ClientContext.UserKey("AmyWiles").Country(StrategyAttributeCountryName.Unitedkingdom).Attr("city", "london").Build();
-      Console.WriteLine("Ready to close");
-      Console.ReadKey();
+      do
+      {
+        fh.LogAnalyticEvent("c-sharp-console");
+        Console.Write("Press a Key");
+      } while (Console.ReadLine() != "x");
+
+
+      // Console.Write("Press a key");
```

Comment:
are these lines deliberately commented out?

Reply:
yes, i didn't want them triggering. Its an experimental example - we need a proper one in the featurehub-examples repo.